pkgs/applications/networking/browsers/amfora/default.nix

```diff
@@ -13,6 +13,12 @@ buildGoModule rec {
 
   vendorSha256 = "10f3bh3r3jc1185r8r1ihg8rprdpl8qwg5b6wqwsda96ydkbpi2b";
 
+  postInstall = ''
+    mkdir -p $out/share/applications
+    sed -i "s:amfora:$out/bin/amfora:" amfora.desktop
```

Comment:
Why you need full path to amfora in `Exec`?

Reply:
I found that a common pattern when packaging desktop files, so that's why

Reply:
I think it's a good change, incase it's not on the user's PATH, but still picked up the desktop file